lib/fullcontact/request.rb

```diff
@@ -10,10 +10,16 @@ def get(path, options={}, raw=false, faraday_options={})
 
     # Perform an HTTP request
     def request(method, path, options, raw=false, faraday_options={})
-      options[:apiKey] = FullContact.options[:api_key]
+      if FullContact.options[:auth_type] == :query
```

Comment:
I tend to prefer guard clauses over nesting, but guard clause makes the line too long here.